plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -314,7 +315,7 @@ func TestWarningResponse(t *testing.T) {
 	assert.NotNil(t, m)
 }
 
-func TestGetRoundTripper(t *testing.T) {
+func TestGetRoundTripperTlsConfig(t *testing.T) {
```

Comment:
@albertteoh How does this look as solution for this [discussion](https://github.com/jaegertracing/jaeger/pull/4247#discussion_r1112874256) ?

Reply:
Yup, this works too.

```suggestion
func TestGetRoundTripperTLSConfig(t *testing.T) {
```